salt/modules/pkgin.py

```diff
@@ -192,7 +192,7 @@ def latest_version(*names, **kwargs):
                 s = _splitpkg(p[0])
                 if s:
                     if not s[0] in pkglist:
-                        if len(p) > 1 and p[1] == '<' or p[1] == '':
+                        if len(p) > 1 and p[1] in ('<', '', '='):
```

Comment:
``p[1]`` is an empty string if the package is not installed and '=' if it is already installed and the latest version.